src/test/java/io/strimzi/kafka/bridge/amqp/AmqpSinkBridgeEndpointMockTest.java

```diff
@@ -5,43 +5,43 @@
 
 package io.strimzi.kafka.bridge.amqp;
 
-import io.strimzi.kafka.bridge.SinkTopicSubscription;
-import io.strimzi.kafka.bridge.converter.MessageConverter;
-import io.vertx.core.AsyncResult;
-import io.vertx.core.Handler;
-import io.vertx.core.Vertx;
-import io.vertx.kafka.client.common.PartitionInfo;
-import io.vertx.kafka.client.common.TopicPartition;
+//import io.strimzi.kafka.bridge.SinkTopicSubscription;
+//import io.strimzi.kafka.bridge.converter.MessageConverter;
+//import io.vertx.core.AsyncResult;
+//import io.vertx.core.Handler;
+//import io.vertx.core.Vertx;
+//import io.vertx.kafka.client.common.PartitionInfo;
+//import io.vertx.kafka.client.common.TopicPartition;
```

Comment:
Remove all the comment out lines.

Reply:
See comment above about AMQP mock tests.